src/unit_tests/win32/test_win_utils.c

```diff
@@ -0,0 +1,164 @@
+/*
+ * Copyright (C) 2015-2021, Wazuh Inc.
+ *
+ * This program is free software; you can redistribute it
+ * and/or modify it under the terms of the GNU General Public
+ * License (version 2) as published by the FSF - Free Software
+ * Foundation.
+ */
+
+#include <stdarg.h>
+#include <stddef.h>
+#include <setjmp.h>
+#include <cmocka.h>
+#include "../../client-agent/agentd.h"
+#include "../wrappers/externals/cJSON/cJSON_wrappers.h"
+#include "../wrappers/common.h"
+#include "../../data_provider/include/sysInfo.h"
+#include "../wrappers/wazuh/shared/debug_op_wrappers.h"
+
+
+#ifdef TEST_WINAGENT
+
+#define TIME_INCREMENT ((time_t)(60))
+
+extern sysinfo_networks_func sysinfo_network_ptr;
+extern sysinfo_free_result_func sysinfo_free_result_ptr;
+
+static agent global_config = { .main_ip_update_interval = (int)TIME_INCREMENT };
+static int test_case_selector = 0;
+static int error_code_sysinfo_network = 0;
+
+int mock_sysinfo_networks_func(cJSON** object) {
+
+    char* ip_update_success = {"{ \"iface\": [ { \"gateway\":\"mock_gateway\", \"IPv4\": { \"address\":\"111.222.333.444\" } } ] }"};
+    char* iface_bad_name = {"{\"iface_fail\":[]}"};
+    char* iface_no_elements = {"{\"iface\":[]"};
+    char* gateway_unknown = {"{ \"iface\": [ { \"gateway\":\"unknown\" } ] }"};
+    char* json_string = 0;
```

Comment:
Some minor suggestions for this set of strings.
- Defining them as `static` will force the compiler to define a pointer to the string in a way similar to how it would deal with a global pointer instead of creating all of them whenever we reach this function.
- Using `const` helps to ensure people who re-read this code later that the strings will not be changed.
- When initializing a pointer, `NULL` is preferred to `0`, since it makes it clearer to read.
```suggestion
    static const char* ip_update_success = "{ \"iface\": [ { \"gateway\":\"mock_gateway\", \"IPv4\": { \"address\":\"111.222.333.444\" } } ] }";
    static const char* iface_bad_name = "{\"iface_fail\":[]}";
    static const char* iface_no_elements = "{\"iface\":[]";
    static const char* gateway_unknown = "{ \"iface\": [ { \"gateway\":\"unknown\" } ] }";
    char* json_string = NULL;
```